### PR TITLE
Allow plan creation with new

### PR DESCRIPTION
**Pull request by barberj**
Currently creating a plan with :new raises a NoMethodError.

[Review on Reviewable](https://reviewable.io/reviews/sb8244/rspec-stripe/1)

### CODE REVIEW

File: lib/rspec-stripe/factories/plan.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ module RSpecStripe::Factory
     def get
       @get ||= begin
         @should_delete = false
-        Stripe::Plan.retrieve(id)
+        Stripe::Plan.retrieve(id.to_s)
```
Comment (sb8244) [inline lib/rspec-stripe/factories/plan.rb:6]: Any idea why this causes NoMethodError? Is there a trace?

Comment (barberj) [inline lib/rspec-stripe/factories/plan.rb:6]: it tries to execute `b` on it...

```
    Failure/Error: Unable to find matching line from backtrace
     NoMethodError:
       undefined method `b' for :new:Symbol
     # /opt/rubies/2.2.2/lib/ruby/2.2.0/cgi/util.rb:9:in `escape'
     # /opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/stripe-1.25.0/lib/stripe/api_resource.rb:20:in `url'
     # /opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/stripe-1.25.0/lib/stripe/api_resource.rb:24:in `refresh'
     # /opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/stripe-1.25.0/lib/stripe/api_resource.rb:31:in `retrieve'
     # /opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-stripe-0.0.6/lib/rspec-stripe/factories/plan.rb:6:in `get'
     # /opt/rubies/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-stripe-0.0.6/lib/rspec-stripe/runner.rb:11:in `call!'
```

**Comment by barberj** [comment]
@sb8244 how goes the investigation?

**Comment by barberj** [comment]
@sb8244 how bouts it?

**Comment by barberj** [comment]
**cough**, **cough**

**Comment by barberj** [comment]
:crying_cat_face:

**Comment by sb8244** [comment]
https://github.com/sb8244/rspec-stripe/commit/c504e0d3d813d55da740da525ea11c2e63db01c5

Were some of those other PRs still needed?

**Pull request merged**